src/Task/Assets/Less.php

```diff
@@ -0,0 +1,179 @@
+<?php
+namespace Robo\Task\Assets;
+
+use Robo\Result;
+use Robo\Task\BaseTask;
+
+/**
+ * Compiles less files.
+ *
+ * ```php
+ * <?php
+ * $this->taskLess([
+ *     'less/default.less' => 'css/default.css'
+ * ])
+ * ->run();
+ * ?>
+ * ```
+ *
+ * Use one of both less compilers in your project:
+ *
+ * ```
+ * "leafo/lessphp": "~0.5",
+ * "oyejorge/less.php": "~1.5"
+ * ```
+ *
+ * You can implement additional compilers by extending this task and adding a
+ * method named after them and overloading the lessCompilers() method to
+ * inject the name there.
+ */
+class Less extends BaseTask
+{
+    /**
+     * The less compiler to use.
+     *
+     * @var string
+     */
+    protected $lessCompiler = 'less';
```

Comment:
I think a mapping by name is better because it allows you to extend the less task and implement your own parser method. For example you want to do something special with one of the compilers. Right now we just use the basics of them but I've seen they provide more functionality. If you only use the class name it wont give you the ability to implement any custom logic around your compiler.